Main fiber from VM 0x1 panicked: 'Invalid argument'
stack traceback:
	input.lua:10: in main chunk
